coderd/autobuild/lifecycle_executor_test.go

```diff
@@ -705,3 +893,68 @@ func mustWorkspaceParameters(t *testing.T, client *codersdk.Client, workspaceID
 func TestMain(m *testing.M) {
 	goleak.VerifyTestMain(m)
 }
+
+func mockEnterpriseTemplateScheduler(t *testing.T) schedule.TemplateScheduleStore {
```

Comment:
It looks like you've copied enterprise code into an AGPL package... you should probably just move the tests that require this to an enterprise package

Reply:
Yeah it's just unfortunate to have autobuild-related tests living outside the package simply because it falls under enterprise functionality